### PR TITLE
[AUD-1685] Optimize AddToPlaylist and AudioBeakdown drawers

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Delays render for AddToPlaylist and AudioBreakdown drawers

Also optimizes WebApp which rerendered a ton due to a boolean

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1685-migrate-other-drawers

File: packages/web/src/common/store/ui/add-to-playlist/actions.ts [hunk 1]
```diff
@@ -8,7 +8,9 @@ export const CLOSE = 'ADD_TO_PLAYLIST/CLOSE'
 
 export const requestOpen = createCustomAction(
   REQUEST_OPEN,
-  (trackId: ID, trackTitle: string) => ({ trackId, trackTitle })
+  (trackId: ID, trackTitle: string) => {
```
Comment (Kyle-Shanks) [inline packages/web/src/common/store/ui/add-to-playlist/actions.ts:11]: linter or do you like explicit returns? lol
If so should change the one below for consistency

**Review by Kyle-Shanks — APPROVED** [review]

File: packages/web/src/common/store/ui/add-to-playlist/actions.ts (hunk 1 above)
Comment (dylanjeffers) [reply packages/web/src/common/store/ui/add-to-playlist/actions.ts:11]: I think I added a console and then didn't clean up, updating :)

File: packages/mobile/src/components/add-to-playlist-drawer/AddToPlaylistDrawer.tsx [hunk 2]
```diff
@@ -36,17 +34,17 @@ export const AddToPlaylistDrawer = () => {
...
+  const { onClose } = useDrawerState('AddToPlaylist')
   const trackId = useSelectorWeb(getTrackId)
   const trackTitle = useSelectorWeb(getTrackTitle)
   const user = useSelectorWeb(getAccountWithOwnPlaylists)
   const [isDrawerGestureSupported, setIsDrawerGestureSupported] = useState(true)
 
-  const handleClose = useCallback(() => {
-    dispatchWeb(close())
-  }, [dispatchWeb])
+  console.log('rendering null')
```
Comment (Kyle-Shanks) [inline packages/mobile/src/components/add-to-playlist-drawer/AddToPlaylistDrawer.tsx:43]: log

File: packages/mobile/src/components/web/WebApp.tsx [hunk 3]
```diff
@@ -561,7 +558,7 @@ const useSplashScreenKey = (dappLoaded: boolean) => {
 }
 
 const mapStateToProps = (state: AppState) => ({
-  state,
+  dappLoaded: state.lifecycle.dappLoaded,
```
Comment (sliptype) [inline packages/mobile/src/components/web/WebApp.tsx:561]:
```suggestion
  dappLoaded: getDappLoaded(state)
```

**Pull request merged**